configs/benchmark/templates/wrk2.yaml

```diff
@@ -16,6 +16,8 @@ spec:
         app: wrk2-prometheus
         custom-affinity: load-generator-node
     spec:
+      nodeSelector:
+        role: benchmark
       restartPolicy: Never
       tolerations:
       - effect: NoSchedule
```

Comment:
I think we can remove the toleration here if we use node labels instead of taints to target nodes for specific workloads.

Reply:
Removed the toleration :-) Thanks for catching it.